### DISCUSSION TITLE
nf-configs repo

### DISCUSSION THREAD

**Discussion by gregdenay** [bio-raum/FooDMe2#61] [General]
@marchoeppner I just noticed there is currently no option to raise issues in the nf-config repo.
I'd like to add a direct link to raise a new issue in the FooDMe2 doc so that users can get in touch directly to have a profile there.

Could you do the following steps (you're the only admin on the repo):

-    Go to the Settings page of your fork.
-    Click 'General' tab on the left
-    Scroll down to the Features subsection
-    Tick the Issues checkbox

**Comment by marchoeppner** [comment]
Done - and you are now admin on that repo as well
